### DISCUSSION TITLE
Incompatibility with GAL PLA replacement?

### DISCUSSION THREAD

**Discussion by nexgen78** [frntc/Sidekick64#99] [Q&A]
Hi,
Have some issues with my original PLA chip, and ordered a GAL chip replacement.
Like this one:
https://www.pcbway.com/project/shareproject/Commodore_64_PLA_replacement.html

With this I only get a black (or somethimes grey screen), when booting with my Sidekick64 inserted. 
Works fine with the original chip though. 
Any idea to why this happens, and any workarounds are welcome. 

Thanks in advance.

**Comment by nosyn00b** [comment]
I currently use my self-built SK64 in combination with a self-built GAL-PLA and have no issues, so I'd say there is not a general GAL-PLA incompatibility.
